src/js/injection_script.js

```diff
@@ -2619,6 +2622,17 @@ domainBypass('apkadmin.com', () => {
   })
 })
 
+domainBypass("bowfile.com", () => {
+	const regex=/.*let next = "(http[^"]+)";.*/
```

Comment:
Is this next variable top-level or not? If it's top level you can just do safely navigate(next)

Reply:
It is not top level, it's inside a function.

Reply:
Ah gotcha!